osf/management/commands/update_bepress_version.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import unicode_literals
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from scripts import utils as script_utils
+from osf.models import PreprintProvider, Subject
+from website.preprints.tasks import on_preprint_updated
+
+logger = logging.getLogger(__name__)
+
+BEPRESS_PROVIDER = PreprintProvider.objects.filter(_id='osf').first()
+BEPRESS_CHANGES = {
+    'create': [
+        'Education:Adult and Continuing Education',
+        'Education:Disability and Equity in Education:Gender Equity in Education',
+        'Education:Early Childhood Education',
+        'Education:Elementary Education',
+        'Education:Indigenous Education',
+        'Education:Language and Literacy Education',
+        'Education:Outdoor Education',
+        'Education:Secondary Education',
+        'Education:Vocational Education',
+        'Engineering:Aviation:Aviation and Space Education',
+        'Medicine and Health Sciences:Medical Education:Interprofessional Education',
+        'Medicine and Health Sciences:Medical Specialties:Tropical Medicine',
+        'Medicine and Health Sciences:Medical Specialties:Sleep Medicine',
+        'Law:Cultural Heritage Law',
+        'Life Sciences:Plant Sciences:Bryology',
+        'Life Sciences:Research Methods in Life Sciences',
+        'Life Sciences:Research Methods in Life Sciences:Animal Experimentation and Research',
+        'Physical Sciences and Mathematics:Earth Sciences:Speleology',
+        'Social and Behavioral Sciences:Food Studies',
+        'Social and Behavioral Sciences:Psychology:Comparative Psychology',
+        'Social and Behavioral Sciences:Psychology:Transpersonal Psychology',
+        'Social and Behavioral Sciences:Public Affairs, Public Policy and Public Administration:Terrorism Studies'
+    ],
+    'rename': {
+        'Native American Studies': 'Indigenous Studies',
+        'Computer Security': 'Information Security',
+        'Military Studies': 'Military and Veterans Studies',
+    }
+}
+
+def update(dry_run=False):
+    for hier in BEPRESS_CHANGES['create']:
+        new_text = hier.split(':')[-1]
+        bepress_parent = BEPRESS_PROVIDER.subjects.get(text=hier.split(':')[-2])
+        logger.info('Creating osf - {}'.format(new_text))
+        bepress_subject = Subject.objects.create(parent=bepress_parent, provider=BEPRESS_PROVIDER, text=new_text)
+        for custom_parent in bepress_parent.aliases.all():
+            if custom_parent.children.exists() or not bepress_parent.children.count() > 1:
```

Comment:
I'm not sure I understand the rationale for `or not bepress_parent.children.count() > 1`. If the bepress subject has 1 child, then the custom parent gets the new subjects. If the bepress subject has 2 children, then the custom parent would not get the new subjects. Is this intended?

Reply:
Yes. The `bepress_parent` has a minimum of 1 child at this point, because the `bepress_subject`, which is the child, was just created. If there are 2, it existed before, but was not included in the `custom_parent`'s children when the custom taxonomy was populated.

Reply:
OK, thanks for the clarification.